Testing/VelaptorTests/NativeInterop/OpenGL/OpenGLServiceTests.cs

```diff
@@ -348,7 +367,7 @@ public void LabelShader_WhenInvoked_LabelsShader()
         service.LabelShader(123, label);
 
         // Assert
-        this.mockGLInvoker.Verify(m => m.ObjectLabel(GLObjectIdentifier.Shader, 123, (uint)label.Length, label));
+        this.mockGLInvoker.Received().ObjectLabel(GLObjectIdentifier.Shader, 123, (uint)label.Length, label);
```

Comment:
Please refactor the `Received()` method to take the parameter value of `1`.

```suggestion
        this.mockGLInvoker.Received(1).ObjectLabel(GLObjectIdentifier.Shader, 123, (uint)label.Length, label);
```